x-pack/plugins/security_solution/server/lib/risk_engine/schema/risk_score_apis.yml

```diff
@@ -44,6 +44,56 @@ paths:
                 $ref: '#/components/schemas/RiskScoresPreviewResponse'
         '400':
           description: Invalid request
+  /engine/status:
```

Comment:
Nice to see the new APIs defined in OpenAPI already! At some point, either in this PR or a follow up, could you or @rylnd move these OpenAPI specs to `/common/api/risk_engine/<route name>` similar to the other API domains? This way the code generation script will be able to find these schemas and we'll have all our OpenAPI schemas in one folder (`/common/api`).